nautobot_chatops/dispatchers/mattermost.py

```diff
@@ -403,11 +407,14 @@ def send_blocks(self, blocks, callback_id=None, modal=False, ephemeral=False, ti
             self.send_exception(mm_error)
 
     @BACKEND_ACTION_SNIPPET.time()
-    def send_snippet(self, text, title=None):
+    def send_snippet(self, text, title=None, ephemeral=None):
         """Send a longer chunk of text as a file snippet."""
         channel = [self.context.get("channel_id")]
         logger.info("Sending snippet to %s: %s", channel, text)
-        self.mm_client.chat_post_message(channel_id=self.context.get("channel_id"), snippet=text)
+        if ephemeral:
+            self.mm_client.chat_post_ephemeral(channel_id=self.context.get("channel_id"), message=text)
```

Comment:
I don't agree with this line. If sending ephemeral, then it no longer sends as a snippet. This means that it is limited to the message size and could break some commands with larger tables.

Reply:
So a limiter would need to be setup? Looks like the max message size is 16,383 chars.

https://mattermost.com/blog/mattermost-5-0-intercept-and-modify-posts-advanced-permissions-longer-posts-and-more/#:~:text=Increased%20character%20limits%20on%20posts,better%20Markdown%20formatting%2C%20including%20tables.

Reply:
How would you want to handle this within the MM client @smk4664 ?

Reply:
My concern is that this function is used specifically to send a snippet, but then you are removing that if it is ephemeral. I would add the snippet option to the chat_post_ephemeral.  I am not sure if you can, but it is worth testing.

Reply:
I think I have a solution in the latest.